tests/zopen_check_install

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+#
+# Basic 'zopen install' check
+#
+
+WORKDIR="$1"
+INCLUDEDIR="$2"
+zopen_tool_binary=`whence zopen`
+zopen_tool_directory=$( dirname ${zopen_tool_binary} )
+
+# Display banner
+${INCLUDEDIR}/banner.sh "tests/check_install"
```

Comment:
minor: I wonder if you can have a different banner for the individual tests, maybe not quite as big? seeing the zOS ASCII (or EBCDIC) art for every test may be too much text.